src/webview/static/base.css

```diff
@@ -484,12 +484,13 @@ time {
 }
 
 .message_inline_image {
-  text-align: center;
+  margin-top: 5px;
 }
 .message_inline_image img,
 .message_inline_ref img,
 .twitter-image img {
-  width: 100%;
+  width: auto; 
+  max-width: 100%; 
```

Comment:
What's the effect of this change? It doesn't seem to be described in the commit message.

Reply:
Hmm...  a change I should have written description of it, but as far as I remember this is the main logic for left alignment of image.

I tried with `float:left` first(just like in web) but it didn't work(🤔 ) and upon using `max-width:100%` or `width:100%` it gave desired result but still 1 problem remained,i.e, message reactions got shifted to the row of respective image and the reaction and image both got aligned somewhat like`flex-direction:row` property(weird,..I think because of `<span>` property but I'm not sure why)

So the final solution was to keep `width:auto` and also giving `max-width` as 100% and it worked without the need for `float` property unlike in web.